packages/lib/pod/src/podTypes.ts

```diff
@@ -68,11 +68,29 @@ export const POD_INT_MIN = 0n;
  */
 export const POD_INT_MAX = (1n << 63n) - 1n;
 
+/**
+ * POD value for EddSA (Baby Jubjub) public keys. Such a value is represented as
+ * a hex string of the (32-byte) compressed form of the key.
+ */
+export type PODEdDSAPublicKeyValue = {
+  type: "eddsa-pubkey";
+  value: string;
+};
+
+/**
+ * A useful identifier for simplified JSON serialisation of EdDSA public keys.
+ */
+export const EDDSA_PUBKEY_PREFIX = "pk";
```

Comment:
I'd make this slightly longer to make it more explicit.  "pk" could stand for private rather than public.
It might make sense to use the same string as the `type` above.  The serialization scheme generalizes better if it's always `type:encodedstr`.

Reply:
Makes sense. `sk` would be used for 'private key' (secret key) in contexts where `pk` is usually used, and I thought we were aiming for brevity. I'll make the change.

Reply:
Understood.  Brevity is certainly a goal which makes me of 2 minds here.  I think it's most important that simple things look simple.  A public key is already going to be a long string of digits, so brevity seems less important for readability.  There's an argument for overall size being a goal too, though there are encodings other than JSON which can do a better job at that.